src/plugins/vis_type_table/public/components/table_vis_split.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { memo } from 'react';
+import { EuiTitle } from '@elastic/eui';
+
+import { IInterpreterRenderHandlers } from 'src/plugins/expressions';
+import { TableGroup } from '../table_vis_response_handler';
```

Comment:
Could you clarify what is the benefit of importing as a type in this particular case?
AFAIK babel will filter out any import from a file if it's only used as a type.
Here is the output file in dev mode, where you could notice the lack of that import, which means babel did its job 😃 

![image](https://user-images.githubusercontent.com/31325372/97559272-45451c00-19ee-11eb-9fbc-7be5e0a4bf9a.png)
 
I assume that the feature is helpful for preventing from including ambiguously resolved imports. 
Issues arise when Babel can’t possibly know if a particular import is a type that should be removed or an actual value that should be kept. This can happen when utilizing re-exports.
But here is not the case..

Reply:
oh yes you are right, here babel works fine so I am fine too 😉